### DISCUSSION TITLE
CS1661 and CS1678 errors when returning `Results.NotFound();` in `app.MapGet`

### DISCUSSION THREAD

**Discussion by aaante** [supabase-community/supabase-csharp#156] [Q&A]
I get CS1661 nad CS1678  errors when returning `Results.NotFound();` in `app.MapGet`.

Here's the code:

```
app.MapGet("/databases", async (Supabase.Client client) =>
{
    var response = await client
        .From<Database>()
        .Get();

    var databases = response.Models.FirstOrDefault();

    if (databases is null)
    {
        return Results.NotFound();
    }

    return databases;
});
```

In [video crash from the docs](https://www.youtube.com/watch?v=uviVTDtYeeE) there are no such issues even though return object is identical...

Any ideas?

**Comment by acupofjose** [comment]
That seems like more of an ASP question than a supabase question - I imagine the API has changed a bit for ASP since the video was made... maybe check here? 

https://stackoverflow.com/questions/77270839/some-minimal-api-endpoints-not-returning-values-warning-asp0016-do-not-return
